app/views/editor/rich.html.erb

```diff
@@ -111,6 +111,9 @@
           <p class="ple-help">
             <span class="ple-help-minor">Select an optional main image for your post.</span>
           </p>
+          <p>
+            <input class="form-control input-lg"  type="text" id="alt-text" placeholder="Add alternate text for image" value="<% if @node && @node.main_image.photo_text %><%= @node.main_image.photo_text %><% else %><%= params[:photo_text] %><% end %>"/>
```

Comment:
@Tlazypanda found that the issue is because some notes do not have images hence the nil  error..we can use a safe navigator to cater for the nodes without 'main_images' like so

`input class="form-control input-lg"  type="text" id="alt-text" placeholder="Add alternate text for image" value="<% if @node && @node&.main_image&.photo_text %><%= @node.main_image.photo_text %><% else %><%= params[:photo_text] %><% end %>"/>`

Reply:
Hi @Tlazypanda I will prioritize this coming week so we can collaborate and have it merged.

Reply:
wow this is v cool! can we see a screenshot at some point? Thanks!!